velox/common/file/FileSystems.h

```diff
@@ -45,6 +45,10 @@ class FileSystem {
 
   // Deletes the file at 'path'. Throws on error.
   virtual void remove(std::string_view path) = 0;
+  // Returns true if the file exists.
+  virtual bool exist(std::string_view path) = 0;
+  // Returns the list of files in a path.
+  virtual std::vector<std::string> list(std::string_view path) = 0;
```

Comment:
There can be a lot of files, so an iterator API might scale better. If we document that this method is for testing only, we can keep it as is though.